style/app.css

```diff
@@ -432,6 +432,13 @@
 	text-align: center;
 }
 
+.core-content #list_pbxs_navbar #unassigned_numbers .content #unassigned_select_all{
+	border-top: 1px solid #fff;
+	border-bottom: 1px solid #ccc;
+	cursor: pointer;
+	height: 50px;
+	line-height: 50px;
+}
 .core-content #list_pbxs_navbar #unassigned_numbers .content #unassigned_actions {
```

Comment:
Instead of copy/pasting those properties to you selector above, you can simply add your new selector here:
```suggestion
.core-content #list_pbxs_navbar #unassigned_numbers .content #unassigned_actions,
.core-content #list_pbxs_navbar #unassigned_numbers .content #unassigned_select_all {
```
And move the `cursor` property from the next selector to this one so it gets applies to those two selectors.

Reply:
done